### PR TITLE
Add 7zip, remove Peazip, Keka not FOSS

### PR DESCRIPTION
**Pull request by dngray**
https://deploy-preview-1797--privacytools-io.netlify.com/software/encryption-tools/

Closes: https://github.com/privacytoolsIO/privacytools.io/issues/1782
Closes: https://github.com/privacytoolsIO/privacytools.io/issues/1784

### CODE REVIEW

**Comment by dngray** [comment]
Seems that netlify isn't deploying a preview

**Comment by blacklight447-ptio** [comment]
> Seems that netlify isn't deploying a preview

netlify says this:
`Liquid Exception: Invalid syntax for include tag: title="7 Zip" image="/assets/img/svg/3rd-party/7zip.svg" description="<strong>7-Zip</strong> is a free and open-source file archiver, a utility used to place groups of files within compressed containers known as "archives". On Linux the command-line tool p7zip is used and integrates into various interfaces such as <a href="https://wiki.gnome.org/Apps/FileRoller">FileRoller</a>, <a href="https://github.com/ib/xarchiver">Xarchiver</a>, <a href="https://kde.org/applications/utilities/ark">Ark</a> and <a href="https://www.keka.io/">Keka</a>. website="https://7-zip.org" forum="" source="https://sourceforge.net/projects/sevenzip/files/" windows="https://7-zip.org/download.html" linux="https://sourceforge.net/projects/p7zip/files" freebsd="https://www.freshports.org/archivers/p7zip" openbsd="https://sourceforge.net/projects/p7zip" netbsd="https://sourceforge.net/projects/p7zip" Valid syntax: {% include file.ext param='value' param2='value' %} in pages/software/encryption.html
10:28:58 AM: jekyll 3.8.5 | Error:  Invalid syntax for include tag:
10:28:58 AM:   title="7 Zip"
10:28:58 AM:   image="/assets/img/svg/3rd-party/7zip.svg"`

seems like you messed up the syntax somewhere.

**Comment by dngray** [comment]
> seems like you messed up the syntax somewhere.

I guess that's what I get for winging it.

File: _includes/sections/file-encryption.html [hunk 1]
```diff
@@ -38,17 +38,17 @@ <h1 id="encrypt" class="anchor"><a href="#encrypt"><i class="fas fa-link anchor-
...
-  windows="https://www.peazip.org/peazip-64bit.html"
-  linux="https://www.peazip.org/peazip-linux.html"
-  freebsd="https://www.freshports.org/archivers/peazip/"
-  openbsd="https://www.peazip.org/peazip-bsd.html"
-  netbsd="https://www.peazip.org/peazip-bsd.html"
+  title="7 Zip"
+  image="/assets/img/svg/3rd-party/7zip.svg"
+  description='<strong>7-Zip</strong> is a free and open-source file archiver, a utility used to place groups of files within compressed containers. On Linux, MacOS etc. the command-line tool <strong>p7zip</strong> is used and integrates into various interfaces such as <a href="https://wiki.gnome.org/Apps/FileRoller">FileRoller</a>, <a href="https://github.com/ib/xarchiver">Xarchiver</a>, <a href="https://kde.org/applications/utilities/ark">Ark</a> and <a href="https://www.keka.io/">Keka</a>.'
+  website="https://7-zip.org"
+  forum=""
```
Comment (Mikaela) [inline _includes/sections/file-encryption.html]: https://forum.privacytools.io/t/discussion-7-zip/3024

File: _includes/sections/file-encryption.html [hunk 2]
```diff
@@ -38,17 +38,17 @@ <h1 id="encrypt" class="anchor"><a href="#encrypt"><i class="fas fa-link anchor-
...
-  forum="https://forum.privacytools.io/t/discussion-peazip-file-encryption/1534"
-  source="https://osdn.net/projects/peazip"
-  windows="https://www.peazip.org/peazip-64bit.html"
-  linux="https://www.peazip.org/peazip-linux.html"
-  freebsd="https://www.freshports.org/archivers/peazip/"
-  openbsd="https://www.peazip.org/peazip-bsd.html"
-  netbsd="https://www.peazip.org/peazip-bsd.html"
+  title="7 Zip"
+  image="/assets/img/svg/3rd-party/7zip.svg"
+  description='<strong>7-Zip</strong> is a free and open-source file archiver, a utility used to place groups of files within compressed containers. On Linux, MacOS etc. the command-line tool <strong>p7zip</strong> is used and integrates into various interfaces such as <a href="https://wiki.gnome.org/Apps/FileRoller">FileRoller</a>, <a href="https://github.com/ib/xarchiver">Xarchiver</a>, <a href="https://kde.org/applications/utilities/ark">Ark</a> and <a href="https://www.keka.io/">Keka</a>.'
```
Comment (Mikaela) [inline _includes/sections/file-encryption.html]:
```suggestion
  description='<strong>7-Zip</strong> is a free and open-source file archiver, a utility used to place groups of files within compressed containers. On Linux, macOS etc. the command-line tool <a href="http://p7zip.sourceforge.net/"><strong>p7zip</strong></a> is used and integrates into various interfaces such as <a href="https://wiki.gnome.org/Apps/FileRoller">FileRoller</a>, <a href="https://github.com/ib/xarchiver">Xarchiver</a>, <a href="https://kde.org/applications/utilities/ark">Ark</a> and <a href="https://www.keka.io/">Keka</a>.'
```

Comment (Mikaela) [reply _includes/sections/file-encryption.html]: I am not sure about Keka. Are there really not open source options or could it have warning on not being OSS?

File: _includes/sections/file-encryption.html [hunk 3]
```diff
@@ -57,5 +57,5 @@ <h3>Worth Mentioning</h3>
   <li><a href="https://cryptomator.org/">Cryptomator</a> - Free client-side AES encryption for your cloud files. Open source software: No backdoors, no registration. <span class="badge badge-warning" data-toggle="tooltip" title="Cryptomator's mobile apps are not open-source."><a href="https://github.com/cryptomator/cryptomator-android/issues/1#issuecomment-257979375"><i class="fas fa-exclamation-triangle"></i></a></span></li>
   <li><a href="https://gitlab.com/cryptsetup/cryptsetup/">Linux Unified Key Setup (LUKS)</a> - A full disk encryption system for Linux using dm-crypt as the disk encryption backend. Included by default in Ubuntu. Available for Windows and Linux.</li>
   <li><a href="https://hat.sh/">Hat.sh</a> - A cross-platform, serverless JavaScript web application that provides secure file encryption using the AES-256-GCM algorithm in your browser. It can also be downloaded and run offline.</li>
-  <li><a href="https://www.keka.io/">Keka</a> - A macOS-only, open-source file archiver with the ability to encrypt files.</li>
+  <li><a href="https://www.keka.io/">Keka</a> - A macOS-only, file archiver with the ability to encrypt files.</li>
```
Comment (Mikaela) [inline _includes/sections/file-encryption.html]: Warn on not being open source?

File: _includes/sections/file-encryption.html (hunk 1 above)
Comment (dngray) [reply _includes/sections/file-encryption.html]: Added

File: _includes/sections/file-encryption.html (hunk 2 above)
Comment (dngray) [reply _includes/sections/file-encryption.html]: Yeah I think I'll remove Keka from there, unless we can find an open source alternative. p7zip can be used on MacOS (it's what I used to do via Homebrew/Macports), but I don't have a Mac now, so I've not kept up to date if there's a modern open-source frontend to p7zip.

File: _includes/sections/file-encryption.html [hunk 4]
```diff
@@ -40,9 +40,9 @@ <h1 id="encrypt" class="anchor"><a href="#encrypt"><i class="fas fa-link anchor-
   include cardv2.html
   title="7 Zip"
   image="/assets/img/svg/3rd-party/7zip.svg"
-  description='<strong>7-Zip</strong> is a free and open-source file archiver, a utility used to place groups of files within compressed containers. On Linux, MacOS etc. the command-line tool <strong>p7zip</strong> is used and integrates into various interfaces such as <a href="https://wiki.gnome.org/Apps/FileRoller">FileRoller</a>, <a href="https://github.com/ib/xarchiver">Xarchiver</a>, <a href="https://kde.org/applications/utilities/ark">Ark</a> and <a href="https://www.keka.io/">Keka</a>.'
+  description='<strong>7-Zip</strong> is a free and open-source file archiver, a utility used to place groups of files within compressed containers. On Linux, MacOS etc. the command-line tool <strong>p7zip</strong> is used and integrates into various interfaces such as <a href="https://wiki.gnome.org/Apps/FileRoller">FileRoller</a>, <a href="https://github.com/ib/xarchiver">Xarchiver</a>, <a href="https://kde.org/applications/utilities/ark">Ark</a>.'
```
Comment (Mikaela) [inline _includes/sections/file-encryption.html]:
```suggestion
  description='<strong>7-Zip</strong> is a free and open-source file archiver, a utility used to place groups of files within compressed containers. On Linux, MacOS etc. the command-line tool <a href="http://p7zip.sourceforge.net/"><strong>p7zip</strong> is used and integrates into various interfaces such as <a href="https://wiki.gnome.org/Apps/FileRoller">FileRoller</a>, <a href="https://github.com/ib/xarchiver">Xarchiver</a>, <a href="https://kde.org/applications/utilities/ark">Ark</a>.'
```

**Pull request merged**